ecosystem/typescript/sdk/src/aptos_account.ts

```diff
@@ -140,6 +142,29 @@ export class AptosAccount {
     return this.signBuffer(toSign);
   }
 
+  /**
+   * Takes source address and seeds and returns the resource account address
+   * @param sourceAddress Address used to derive the resource account
+   * @param seeds The seeds need to be in hexadecimal format
+   * @returns The resource account address
+   */
+
+  getResourceAccountAddress(sourceAddress: MaybeHexString, seeds: string): HexString {
+    const source = bcsToBytes(AccountAddress.fromHex(sourceAddress));
+    const seed = new HexString(seeds).toUint8Array();
+
+    const originBytes = new Uint8Array(source.length + seed.length);
+    this.address();
```

Comment:
huh?

Reply:
if we make it a static method, we can remove `this.address()`. Sorry my bad.